client/signup/config/steps.js

```diff
@@ -262,12 +262,12 @@ export default {
 
 	'creds-confirm': {
 		stepName: 'creds-confirm',
-		providesDependencies: [],
+		providesDependencies: [ 'rewindconfig' ],
```

Comment:
I'm curious why this is necessary here.

Reply:
@rcoll it's to whitelist the dependency, otherwise JS console throws a warning.